src/main/java/com/sprint/mission/discodeit/service/jcf/JCFMessageService.java

```diff
@@ -0,0 +1,127 @@
+package com.sprint.mission.discodeit.service.jcf;
+
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.service.MessageService;
+
+import java.util.ArrayList;
+
+public class JCFMessageService implements MessageService {
+
+    private static JCFMessageService instance = new JCFMessageService();
+    private final ArrayList<Message> data;
```

Comment:
```suggestion
    private final ArrayList<Message> messages;
```
변수명은 정확하게 선언하는게 좋습니다.